### ISSUE TITLE
W503: yapf inserts line break before rather than after an '-' binary operator

### ISSUE THREAD

**Issue by chtseac** [google/yapf#498]
There is no option that I can find for this, unlike #38 and #252.

Code:

```python
import inspect


def get_default_of(func, argname):
    argspec_of_func = inspect.getargspec(func)
    return argspec_of_func.defaults[argspec_of_func.args.index(argname) - len(argspec_of_func.args)]
```

Result:

```python
import inspect


def get_default_of(func, argname):
    argspec_of_func = inspect.getargspec(func)
    return argspec_of_func.defaults[argspec_of_func.args.index(argname)
                                    - len(argspec_of_func.args)]
```

This yields W503 for pyflakes: line break before binary operator.

**Comment by LCLx** [comment]
Yes, that's true. I met the same problem. Yapf inserts a line break after a '-' in '[]'.

**Comment by AWhetter** [comment]
A line break _should_ be before a binary operator. PEP8 was updated to expect binary operators before a line break (https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) and the flake8 warning has since been turned off by default (https://github.com/PyCQA/pycodestyle/pull/499).

**Comment by ssbarnea** [comment]
Am I wrong to assume that this means that current version of yapf is incompatible with current version of pycodestyle? Is there a way to tune yapf to avoid creating files that do not match current pep-0008 requirements?

**Comment by AWhetter** [comment]
They should be compatible. I think this issue can be closed because yapf has correct behaviour that matches PEP8. Also pycodestyle has turned off W503 by default and turned on W504 instead, and so yapf is compatible with pycodestyle.
